sellercenter-attributes/src/main/java/com/gfg/sellercenter/attributes/entity/ProductWithAttributeInformation.java

```diff
@@ -0,0 +1,21 @@
+package com.gfg.sellercenter.attributes.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import org.json.JSONObject;
+import java.util.Map;
+
+@AllArgsConstructor
+@Getter
+@EqualsAndHashCode
+public class ProductWithAttributeInformation {
```

Comment:
Probably it is not possible to make it simplier, but I find overall structure of classes a bit complex (map with map with map). I would be really happy if at least somewhere it could be possible to replace maps with plain lists or arrays.

Reply:
For other reviewers: now I understand that we can't escape current structure of classes. Thanks @Cloudwalker1986 for explanation.